### PR TITLE
Add fade up (FDUP) and fade down (FDDOWN)

### PR DESCRIPTION
**Pull request by boojew**
I've added the basic features needed to access slow fade up and down.

### CODE REVIEW

**Comment by shbatm** [comment]
I think you'd need a FDSTOP too.

**Comment by boojew** [comment]
Good call :)

File: PyISY/Nodes/node.py [hunk 1]
```diff
@@ -274,6 +274,49 @@ def faston(self):
...
+                                           self._id)
+            return False
+        else:
+            self.parent.parent.log.info('ISY turned did a fade down with node: '
+                                        + self._id)
+            self.update(_change2update_interval, hint=0)
+            return True
+
+    def fdstop(self):
+        """ Slow fade down"""
```
Comment (OverloadUT) [inline PyISY/Nodes/node.py]: Should say Fade stop

File: PyISY/Nodes/node.py [hunk 2]
```diff
@@ -274,6 +274,49 @@ def faston(self):
...
+        response = self.parent.parent.conn.nodeFadeStop(self._id)
+
+        if response is None:
+            self.parent.parent.log.warning('ISY could not fade stop node: ' +
+                                           self._id)
+            return False
+        else:
+            self.parent.parent.log.info('ISY turned did a fade stop with node: '
```
Comment (OverloadUT) [inline PyISY/Nodes/node.py]: remove `turned`

File: PyISY/Connection.py [hunk 3]
```diff
@@ -196,6 +196,21 @@ def nodeFastOn(self, nid):
         response = self.request(req_url)
         return response
 
+    def nodeSlowFadeUp(self, nid):
```
Comment (OverloadUT) [inline PyISY/Connection.py]: What's the thinking behind calling it "Slow" fade up and down? If this matches some ISY documentation or something then that's fine, but I thought this was generally just called Fade Up.

**Comment by boojew** [comment]
Thanks. Changes pushed. I need to stop copying and pasting late at night ;)

File: PyISY/Connection.py (hunk 3 above)
Comment (OverloadUT) [reply PyISY/Connection.py]: @boojew any thoughts on this one?

**Comment by shbatm** [comment]
FWIW: in the WSDK it's just Fade, not Slow Fade

| Command | Name |
| - | - |
| FDDOWN | Fade Down |
| FDSTOP | Fade Stop |
| FDUP | Fade Up |

File: PyISY/Connection.py (hunk 3 above)
Comment (shbatm) [reply PyISY/Connection.py]: FWIW: in the WSDK it's just Fade, not Slow Fade

| Command | Name |
| - | - |
| FDDOWN | Fade Down |
| FDSTOP | Fade Stop |
| FDUP | Fade Up |

**Pull request closed without merging**